SEED-platform/seed
# refactor: allow meter data import from PM data requests

#### Any background context you want to provide?
Users of SEED have requested the ability to import meter data from Data Request spreadsheets

#### What's this PR do?
Adds the ability for the existing meter imports flow to import Data Request meter data.

This has two technical implications:
- adding a new `Electricity - Unknown` source type for Meters. Data requests don't specify the type of data
- adding a new meter display type for `Electricity - Unknown`

Note: data requests do not include source meter IDs, so `Unknown` is use for the IDs of meters which come from Data Requests

#### How should this be manually tested?
1. download an Excel workbook from a data request which includes natural gas and/or electricity data. Either use the example file provided by OpenTech, or use SEED's test account and go to reports and for SEED City Test Report click "Download current report in Excel".
2. Make sure one or more of your properties have the appropriate PM ID which will link to the meter data being imported
3. import PM meter data like normally, select the data request file

#### What are the relevant tickets?
#2673 

#### Screenshots (if appropriate)
![Screen Shot 2021-06-14 at 5 06 48 PM](https://user-images.githubusercontent.com/18518728/121970158-286a1880-cd33-11eb-868d-c31be18abcee.png)
